core/Dispatch.cpp

```diff
@@ -239,9 +239,14 @@ namespace olympia
                                               << " not found in dispatchers, make sure pipe is "
                                                  "defined and has an assigned dispatcher");
                 auto & dispatchers = dispatchers_[static_cast<size_t>(target_pipe)];
-                // so we have a map here that checks for which valid dispatchers for that
-                // instruction target pipe map needs to be: "int": [exe0, exe1, exe2]
-                if (target_pipe != InstArchInfo::TargetPipe::LSU)
+                sparta_assert(dispatchers.size() > 0,
+                              "Pipe Target: "
+                                  << target_pipe
+                                  << " doesn't have any execution unit that can handle that target "
+                                     "pipe. Did you define it in the yaml properly?")
+                    // so we have a map here that checks for which valid dispatchers for that
+                    // instruction target pipe map needs to be: "int": [exe0, exe1, exe2]
+                    if (target_pipe != InstArchInfo::TargetPipe::LSU)
```

Comment:
Indention is funny.   I think it's probably the missing semicolon on the assertion

Reply:
Added semicolon, didn't seem to fix formatting too much, this is with clang-format too.